server/src/main/java/org/elasticsearch/action/support/broadcast/BroadcastResponse.java

```diff
@@ -120,4 +146,11 @@ public void writeTo(StreamOutput out) throws IOException {
             exp.writeTo(out);
         }
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        // TODO: move BroadcastResponse building codes from RestActions to itself here
```

Comment:
I know that I told you to add this TODO, but I looked deeper and I don't think we will address this anytime soon. Let's remove it, I think what we do now is fine.